packages/snap/snap.manifest.json

```diff
@@ -29,7 +29,9 @@
       "dapps": true
     },
     "snap_manageAccounts": {},
-    "snap_manageState": {}
+    "endowment:page-home": {},
```

Comment:
are we able to add allowedOrigins to these endowments?

Reply:
[05025d5](https://github.com/MetaMask/snap-watch-only/pull/2/commits/05025d57937851bb3d40c697916f8452e816218b)